Cargo.toml

```diff
@@ -271,8 +271,8 @@ indexmap = { version = "~1.9.3", default-features = false, features = ["serde"]
 infer = { version = "0.13.0", default-features = false, optional = true}
 indoc = { version = "2.0.1", default-features = false }
 inventory = { version = "0.3.5", default-features = false }
-k8s-openapi = { version = "0.16.0", default-features = false, features = ["api", "v1_19"], optional = true }
-kube = { version = "0.75.0", default-features = false, features = ["client", "native-tls", "runtime"], optional = true }
+k8s-openapi = { version = "0.18.0", default-features = false, features = ["api", "v1_26"], optional = true }
+kube = { version = "0.82.0", default-features = false, features = ["client", "openssl-tls", "runtime"], optional = true }
```

Comment:
This is interesting to see @jszwedko - that feature was removed by [this PR](https://github.com/kube-rs/kube/pull/1044#issue-1401639151). Worth noting given our previous conversations around openssl vs native.